pom.xml

```diff
@@ -71,7 +71,7 @@
         <dep.pinot.version>0.11.0</dep.pinot.version>
         <dep.druid.version>30.0.1</dep.druid.version>
         <dep.jaxb.version>2.3.1</dep.jaxb.version>
-        <dep.hudi.version>0.14.0</dep.hudi.version>
+        <dep.hudi.version>0.15.0</dep.hudi.version>
```

Comment:
I think even if this is upgraded to [1.0.2](https://mvnrepository.com/artifact/org.apache.hudi/hudi-presto-bundle/1.0.2), we would still not be addressing the [high](https://github.com/advisories/GHSA-53wx-pr6q-m3j5) and [critical](https://nvd.nist.gov/vuln/detail/CVE-2025-30065) CVEs arising from parquet-avro